cypress/integration/1-getting-started/todo.spec.js

```diff
@@ -0,0 +1,143 @@
+/// <reference types="cypress" />
+
+// Welcome to Cypress!
```

Comment:
You might want to consider adding the cypress examples to .gitignore so they aren't picked up by the repository, or deleting them entirely - as most developers won't be interested in seeing these example tests.